enable/savage/trait_defs/ui/__init__.py

```diff
@@ -0,0 +1,93 @@
+# Import hooks for loading enable.savage.trait_defs.ui.qt.* in place of 
```

Comment:
@homosapien-lcy this feels like a lot of duplicated code. `enable` depends on `pyface`, no? I suspect we could make sure it depends on pyface 8.x and just import the code from there.

Reply:
Yeah, currently the issue is, if the pyface version the user need is not 8.x, we would still need the ShadowedModuleFinder in Enable since we cannot import it. Do you want the code to first try to import from the ShadowModuleFinder like this?:

```
try:
    from pyface.ui import ShadowedModuleFinder
except ImportError:
    from enable.savage.trait_defs.ui import ShadowedModuleFinder
```

Reply:
It's good to think of backward compatibility. The current enable version has a requirement of pyface >= 7.2 (see https://github.com/enthought/enable/blob/main/enable/__init__.py#L19). You can bump the requirement for this release to be >= 8.0.